### PR TITLE
Seo upgrade 

### PR DESCRIPTION
**Pull request by thomas-mauran**
## How should this be manually tested?

  - Step 1 : merge and build with actions
  - Step 2 : check titles and meta on the official doc core

### CODE REVIEW

**Comment by kuzzle** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Found some issues !</td>
    </tr>
  </tbody>
</table>



## Executed Checks

### :heavy_check_mark: Branch Merge Check



### :x: Changelog Tag Check

Pull Requests must have a `changelog:XXX` label.
If you want to allow this PR to not have a changelog tag, add the following line to the PR body:


```
ci-no-changelog-tag
```

### :heavy_check_mark: PR Body Check


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5069eecef8e125fed667d2f50f36e1e382b4e696
</p>

File: doc/2/guides/write-plugins/old-guides/index.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
...
-description: Kuzzle Old Plugins guides
+title: Old Guides | Write plugins | Guide 
+meta:
+  - name: description
+    content:  Kuzzle Old Plugins guides
+  - name: keywords
+    content: Kuzzle, Documentation, kuzzle write pluggins, old guides
 ---
+
+<Redirect to="hooks" />
```
Comment (rolljee) [inline doc/2/guides/write-plugins/old-guides/index.md:13]: Is this the write redirect ?

**Review by rolljee — APPROVED** [review]

**Review by OlivierCavadenti — COMMENTED** [review]
I think it's not good to do a merge from 2-dev in master and add commits in between.
IMO you need to do a dedicated branch to merge on 2-dev, then master, no ?

File: doc/2/guides/write-plugins/old-guides/index.md (hunk 1 above)
Comment (thomas-mauran) [reply doc/2/guides/write-plugins/old-guides/index.md:13]: Yes, I doubled checked in the code and on the pre prod and it's redirecting on hooks 
https://next-docs.kuzzle.io/core/2/guides/write-plugins/old-guides

**Pull request closed without merging**